docs/sdk/api/getCredentials.md

```diff
@@ -0,0 +1,46 @@
+# `get-credentials`
+Retrieve commercetools platform credentials from environment variables or file system.
+
+## Install
+
+```bash
+npm install --save @commercetools/get-credentials
+```
+
+## Usage
+
+First `get-credentials` will try to load the credentials from a file `ct-credentials.env` in the current directory and then from the `/etc` directory. If a project's credentials exist in both files the values from the current directory are used. Secondly credentials will be retrieved from an environment variable containing the project key: `CT_<project key>` containing `<client id>:<client secret>`. Notice that a project key will be uppercased and dashes will be replaced by an underscore. So for example: `my-projectkey` becomes `MY_PROJECTKEY`.
```

Comment:
It doesn't support JSON and no home directory either. To keep things simple I want to minimize the amount of locations the config can be and have one format. So it looks in the working directory and in `/etc` which is an appropriate location for config files.